plugins/obs-qsv11/obs-qsv11-plugin-main.c

```diff
@@ -58,6 +58,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-qsv11", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "Intel QuickSyncVideo H.264 encoder support for Windows";
```

Comment:
The correct style is "Intel® Quick Sync Video".

Reply:
I know intel likes to use (R), but by the same token every company name and branded technology could have some kind of legal marker. Will split the Quick Sync Video part.